<launch>

    <!-- BEGIN ROBOT 1 -->
    <group ns="robot1">
        <param name="tf_prefix" value="robot1_tf"/>
        <include file="$(find multiple_turtlebots_estm)/launch/one_robot.launch">
            <arg name="init_pose" value="-x 2 -y -0 -z 0 -Y 1.5707"/>
            <arg name="robot_name" value="Robot1"/>
        </include>
    </group>

    <!-- BEGIN ROBOT 2 -->
    <group ns="robot2">
        <param name="tf_prefix" value="robot2_tf"/>
        <include file="$(find multiple_turtlebots_estm)/launch/one_robot.launch">
            <arg name="init_pose" value="-x 4 -y 0 -z 0 -Y 1.5707"/>
            <arg name="robot_name" value="Robot2"/>
        </include>
    </group>

    <!-- BEGIN ROBOT 3 -->
    <!-- <group ns="robot3">
        <param name="tf_prefix" value="robot3_tf"/>
        <include file="$(find multiple_turtlebots_estm)/launch/one_robot.launch">
            <arg name="init_pose" value="-x 0 -y 2 -z 0 -Y 3.14"/>
            <arg name="robot_name" value="Robot3"/>
        </include>
    </group> -->

</launch>
